content/03.categorize.md

```diff
@@ -197,13 +191,16 @@ Of particular interest, when the authors first trained a neural network to predi
 This demonstrates how deep learning methods can repurpose features across tasks, improving overall predictions as the field tackles new challenges.
 The Discussion further reviews this type of transfer learning.
 
-Several authors have created reusable feature sets for medical terminologies using natural language processing and neural embedding models, as popularized by Word2vec [@tag:Word2Vec].
+Several authors have created reusable feature sets for medical terminologies using natural language processing and neural embedding models, as popularized by word2vec [@arxiv:1301.3781].
+Minarro-Giménez et al. [@doi:10.3233/978-1-61499-432-9-584] applied the word2vec deep learning toolkit to medical corpora and evaluated the efficiency of word2vec in identifying properties of pharmaceuticals based on mid-sized, unstructured medical text corpora without any additional background knowledge. 
 A goal of learning terminologies for different entities in the same vector space is to find relationships between different domains (e.g. drugs and the diseases they treat).
 It is difficult for us to provide a strong statement on the broad utility of these methods.
 Manuscripts in this area tend to compare algorithms applied to the same data but lack a comparison against overall best-practices for one or more tasks addressed by these methods.
-Techniques have been developed for free text medical notes [@doi:10.1145/2661829.2661974], ICD and National Drug Codes, and claims data [@doi:10.1145/2939672.2939823].
+Techniques have been developed for free text medical notes [@doi:10.1145/2661829.2661974], ICD and National Drug Codes [@doi:10.18653/v1/w17-2342; @tag:world2004international], and claims data [@doi:10.1145/2939672.2939823].
 Methods for neural embeddings learned from electronic health records have at least some ability to predict disease-disease associations and implicate genes with a statistical association with a disease [@doi:10.1038/srep32404].
-However, the evaluations performed did not differentiate between simple predictions (i.e. the same disease in different sites of the body) and non-intuitive ones. While promising, a lack of rigorous evaluations of the real-world utility of these kinds of features makes current contributions in this area difficult to evaluate.
+However, the evaluations performed did not differentiate between simple predictions (i.e. the same disease in different sites of the body) and non-intuitive ones. 
+Moreover, Jagannatha and Yu  [@pmid:27885364] employed a bidirectional LSTM structure to extract adverse drug events from electronic health records, and Lin et al. [@doi:10.18653/v1/w17-2341] investigated using CNN to extract temporal relations.
```

Comment:
There are so many leading clauses here ("However," "Moreover," "While promising" and "To examine the true utility,") that it was breaking up the flow. Can you keep the at most two that you think are most important and remove the others? This is a minor point, but it'll save time later when we do final revisions for flow.